bot/core.py

```diff
@@ -96,7 +96,8 @@ async def process_commands(self, message: discord.Message, /):
 
     async def on_app_command_error(self, interaction: "InteractionType", error: app_commands.AppCommandError):
         """Handle errors in app commands."""
-        if isinstance(error, IgnorableException):
+
+        if isinstance(error.__cause__, IgnorableException):
```

Comment:
Should we always check `error.__cause__`?
Could `error.__cause__` not be None in any case?

Reply:
can you please clarify what you are asking, i dont see how else we could check for ignorableexception if not done like this.